crates/hir-def/src/macro_expansion_tests/mbe.rs

```diff
@@ -2029,3 +2029,25 @@ fn f() {
     "#]],
     );
 }
+
+#[test]
+fn lifetime_repeat() {
+    check(
+        r#"
+macro_rules! m {
+    ($($x:expr)'a*) => (stringify!($($x)'b*));
+}
+fn f() {
+    let _ = m!(0 'a 1 'a 2);
+}
+    "#,
+        expect![[r#"
+macro_rules! m {
+    ($($x:expr)'a*) => (stringify!($($x)'b*));
+}
+fn f() {
+    let _ = stringify!(0 'b1 'b2);
```

Comment:
That seems to be incorrect?

Reply:
Fixed the pretty printer used for this test to handle this.